db/collection.go

```diff
@@ -228,12 +206,19 @@ func (db *db) createCollection(
 func (db *db) updateCollection(
```

Comment:
todo: Spotted by Fred - this should no longer be called `updateCollection`.

- [ ] Rename to `updateSchema`

Reply:
Nothing that it should also be moved to `schema.go`

Reply:
I think it should be co-located with the other functions that update the descriptions (including the future patchCollection), spreading those functions out will not aid readability. 

Your thought should be re-visited in https://github.com/sourcenetwork/defradb/issues/1963 once everything in this space makes a little more sense and we can move everything together.